installer/win/LaunchBrackets/LaunchBrackets/LaunchBrackets.cs

```diff
@@ -0,0 +1,30 @@
+using System;
```

Comment:
Copyright missing?

Reply:
@gavankar , please address this.

Reply:
@nethip we have never had a copyright in the installer files so far(installer/win/Brackets.wxs), should we be adding one now?